chain-crypto/src/algorithms/ed25519.rs

```diff
@@ -47,6 +47,9 @@ impl AsymmetricKey for Ed25519 {
     type Secret = Priv;
     type Public = Pub;
 
+    const SECRET_BECH32_HRP: &'static str = "ed25519_secret";
+    const PUBLIC_BECH32_HRP: &'static str = "ed25519_public";
```

Comment:
I don't see any of these new private constants used anywhere?

Reply:
They are not private, traits always define only public things. They're already used in jormungandr branch with key generation in CLI.

Reply:
Ah, sorry, these are associated const items.